source/classes.tex

```diff
@@ -6858,22 +6858,16 @@
 \item
 overload resolution\iref{over.match},
 as applied to \tcode{x <=> y}
-(also considering synthesized candidates with reversed order of parameters\iref{over.match.oper}),
 results in an ambiguity
 or a function that is deleted or inaccessible from the operator function,
 or
 
 \item
-the operator \tcode{@}
-cannot be applied to the return type of \tcode{x <=> y} or \tcode{y <=> x}.
+\tcode{x <=> y @ 0} is ill-formed.
```

Comment:
This fully covers the check in the first bullet, but it also covers some other things (overload resolution can succeed and pick a function that is nonetheless not callable). As such, changing to this wording is also non-editorial. =(

I think we should just drop the "or `y <=> x`" from this bullet for now.

Reply:
Core issue it is then 😃